Geocortex.Mobile.Samples/Geocortex.Mobile.Samples/Samples/AppConfiguration/Commands/commands.xml

```diff
@@ -7,7 +7,7 @@
     https://geocortex.com/layout/v1 ../../../layout-custom.xsd"
     xmlns:custom="https://geocortex/layout/samples">
 
-  <gxm:taskbar>
+  <gxm:taskbar orientation="vertical">
```

Comment:
Why vertical? Don't we want this to be automatic so that we get horizontal on a phone?

Reply:
I think that would typically be the case, yeah. But there's an issue with the way the samples selector pushes the app such that it breaks the map view when pairing iOS with the horizontal task bar. I haven't been able to determine why that's the case. It's been this way for since these samples were crafted. But my suspicion is it's related to some of the iOS navigation stack manipulation that happens mobile side.

Reply:
That's unfortunate. Would you be able to spend a bit of time investigating that? If you can't find a solution in a reasonable time period, then we can do the vertical orientation.

Reply:
Alright, I've pushed a change such that iOS + phone will use PushModalAsync and inject a button so we can still return to the samples. With that as a workaround, we're able to have horizontal task bar, so I've removed the `orientation="vertical"` from some layouts.